### PR TITLE
docs: update app descriptions

### PR DESCRIPTION
**Pull request by pktiuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,10 @@
 
 ## Description
 
-antimicrox is a graphical program used to map keyboard keys and mouse controls to a gamepad. You can use this program to control any desktop application with a gamepad; on Linux, this means that your system has to be running an X environment in order to run this program.
+antimicrox is a graphical program used to map gamepad keys to keyboard, mouse, scripts and macros. You can use this program to control any desktop application with a gamepad on Linux.
```
Comment (gombosg) [inline README.md:18]: I'd transition to the GUI name here:
```suggestion
AntiMicroX is a graphical program used to map gamepad keys to keyboard, mouse, scripts and macros. You can use this program to control any desktop application with a gamepad on Linux. 
```

File: README.md [hunk 2]
```diff
@@ -15,7 +15,10 @@
 
 ## Description
 
-antimicrox is a graphical program used to map keyboard keys and mouse controls to a gamepad. You can use this program to control any desktop application with a gamepad; on Linux, this means that your system has to be running an X environment in order to run this program.
+antimicrox is a graphical program used to map gamepad keys to keyboard, mouse, scripts and macros. You can use this program to control any desktop application with a gamepad on Linux. 
+It can be also used for generating SLD2 configuration (useful for mapping atypical gamepads to generic ones like xbox360).
+
+Currently wy don't support Wayland ([#32](https://github.com/AntiMicroX/antimicrox/issues/32)) that your system has to be running an X environment in order to run this program.
```
Comment (gombosg) [inline README.md:21]: typo, also I'd be a tad bit more precise.
```suggestion
Currently we don't support Wayland ([#32](https://github.com/AntiMicroX/antimicrox/issues/32)) - your system has to be running X.org in order to run this program.
```

File: other/appdata/PO_files/en.po [hunk 3]
```diff
@@ -16,7 +16,7 @@ msgstr ""
 
 #. (itstool) path: description/p
 #: io.github.antimicrox.antimicrox.appdata.xml.in:9
-msgid "AntiMicroX is a graphical program used to map keyboard keys and mouse controls to a gamepad. This program is useful for playing PC games using a gamepad that do not have any form of built-in gamepad support."
+msgid "AntiMicroX is a graphical program used for mapping gamepad buttons to keyboard, mouse, scripts and macros."
```
Comment (gombosg) [inline other/appdata/PO_files/en.po:19]: Let's be consistent then :slightly_smiling_face: 
```suggestion
msgid "AntiMicroX is a graphical program used to map gamepad buttons to keyboard, mouse, scripts and macros."
```

File: CMakeLists.txt [hunk 4]
```diff
@@ -682,7 +682,7 @@ if(UNIX)
 if(CPACK_GENERATOR STREQUAL "DEB")
   set(CPACK_DEBIAN_PACKAGE_DEPENDS "qtbase5-dev, libsdl2-2.0-0, libqt5x11extras5, libc6")
   set(CPACK_DEBIAN_PACKAGE_MAINTAINER "pktiuk <kotiuk@zohomail.eu>")
-  set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "AntimicroX is a graphical program used to map keyboard keys and mouse controls to a gamepad. This program is useful for playing PC games using a gamepad that do not have any form of built-in gamepad support. However, you can use this program to control any desktop application with a gamepad.
+  set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "AntiMicroX is a graphical program used for mapping gamepad buttons to keyboard, mouse, scripts and macros.
```
Comment (gombosg) [inline CMakeLists.txt:685]:
```suggestion
  set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "AntiMicroX is a graphical program used to map gamepad buttons to keyboard, mouse, scripts and macros. 
```

File: other/antimicrox.1 [hunk 5]
```diff
@@ -8,7 +8,7 @@ AntiMicroX \- map keyboard keys and mouse controls to a gamepad
 [\fIOPTION\fR] [\fIPROFILE\fR]
 .SH DESCRIPTION
 .PP
-AntiMicroX is a graphical program used to map keyboard keys and mouse controls to a gamepad. This program is useful for playing PC games using a gamepad that do not have any form of built-in gamepad support. However, you can use this program to control any desktop application with a gamepad.
+AntiMicroX is a graphical program used for mapping gamepad buttons to keyboard, mouse, scripts and macros. However, you can use this program to control any desktop application with a gamepad.
```
Comment (gombosg) [inline other/antimicrox.1:11]:
```suggestion
AntiMicroX is a graphical program used to map gamepad buttons to keyboard, mouse, scripts and macros. However, you can use this program to control any desktop application with a gamepad.
```

File: other/appdata/io.github.antimicrox.antimicrox.appdata.xml.in [hunk 6]
```diff
@@ -4,15 +4,23 @@
...
-     controls to a gamepad. This program is useful for playing PC games using a
-     gamepad that do not have any form of built-in gamepad support. 
+     AntiMicroX is a graphical program used to map gamepad buttons/joysticks
+     to keyboard, mouse, scripts and macros.
+     It can be also used for generating SLD2 configuration (useful for mapping 
+     atypical gamepads to generic ones like xbox360).
     </p>
     <p>
-     AntiMicroX was inspired by QJoyPad but has additional features.
+    It allows mapping of gamepads/joystick buttons to:
```
Comment (gombosg) [inline other/appdata/io.github.antimicrox.antimicrox.appdata.xml.in:16]: Please use HTML `<ul>` here, too, it's [supported](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description).

File: other/appdata/io.github.antimicrox.antimicrox.appdata.xml.in [hunk 7]
```diff
@@ -4,15 +4,23 @@
...
+    - keyboard buttons
+    - mouse buttons and moves
+    - scripts and apps
```
Comment (gombosg) [inline other/appdata/io.github.antimicrox.antimicrox.appdata.xml.in:19]:
```suggestion
    - scripts and launching apps
```

**Comment by pktiuk** [comment]
I am not sure about content of `other/appdata/PO_files/en.po` should I also edit this part?

**Review by gombosg — COMMENTED** [review]
Nice! :+1: 
I'd make the main blurb "AntiMicroX is a graphical program used to map gamepad buttons to keyboard, mouse, scripts and macros." the same everywhere.
Feel free to merge after addressing suggestions.

p.s. please use force push sparingly after requesting a review, it can mess things up in my window :slightly_smiling_face:

**Comment by gombosg** [comment]
> I am not sure about content of other/appdata/PO_files/en.po should I also edit this part?
@pktiuk  yes, feel free.

**Comment by pktiuk** [comment]
But that file seems to be autogenerated (lines like `#: io.github.antimicrox.antimicrox.appdata.xml.in:14` pointing at place in file, should I update every line in this file, or `#` is just a comment and I don't need to care about it?

**Comment by gombosg** [comment]
Oh crap yes. It's autogenerated.
```
itstool io.github.antimicrox.antimicrox.appdata.xml.in > ./PO_files/en.po
```
It also puts unneeded entries into the file, but let's just leave those there. It's not an issue, translators will know not to translate URLs.

**Comment by pktiuk** [comment]
I will leave this file as it is.

**Comment by gombosg** [comment]
Please no, let me do it. It may be processed wrong by Flathub or GNOME app store etc.

**Comment by gombosg** [comment]
Done! I updated the correct `itstool` command above. Now it uses the rules file and only outputs items we need.
If translators translate these language files, they will be used in app store applications.

**Review by gombosg — APPROVED** [review]
LGTM now

**Pull request merged**